packages/flx/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ homepage: https://github.com/flutter/flutter/tree/master/packages/flx
 dependencies:
   bignum: ^0.0.7
   asn1lib: ^0.4.1
-  cipher: ^0.7.1
+  pointycastle: ^0.10.0-dev
```

Comment:
I don't think there is an 0.10.0 yet:

> pub get
Resolving dependencies... 
Package pointycastle has no versions that match >=0.10.0 <0.11.0 derived from:
- flx depends on version ^0.10.0